dev/automate_training.py

```diff
@@ -29,48 +31,56 @@
 def get_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("-c", "--config", required=True, help="Base config file path.")
-    parser.add_argument("--all-combin", dest='all_combin', action='store_true', help="To run all combinations of config")
+    parser.add_argument("-n", "--n-iterations", dest="n_iterations",
+                        type=int, help="Number of times to run each config .")
+    parser.add_argument("--all-combin", dest='all_combin', action='store_true',
+                        help="To run all combinations of config")
     parser.set_defaults(all_combin=False)
 
     return parser
 
 
 def worker(config):
     current = mp.current_process()
-    #ID of process used to assign a GPU
+    # ID of process used to assign a GPU
     ID = int(current.name[-1]) - 1
 
-    #Use GPU i from the array specified in the config file
+    # Use GPU i from the array specified in the config file
     config["gpu"] = config["gpu"][ID]
 
-    #Call ivado cmd_train
+    # Call ivado cmd_train
     try:
-        #Save best validation score
-        best_training_dice, best_training_loss, best_validation_dice, best_validation_loss = ivado.cmd_train(config)
+        # Save best validation score
+        best_training_dice, best_training_loss, best_validation_dice, best_validation_loss = ivado.cmd_train(
+            config)
     except:
         logging.exception('Got exception on main handler')
         print("Unexpected error:", sys.exc_info()[0])
         raise
 
     # Save config file in log directory
-    config_copy = open(config["log_directory"] + "/config.json","w")
+    config_copy = open(config["log_directory"] + "/config.json", "w")
     json.dump(config, config_copy, indent=4)
 
     return config["log_directory"], best_training_dice, best_training_loss, best_validation_dice, best_validation_loss
 
 
+def test_worker(config):
+    return config["log_directory"], np.random.normal(0, 1), np.random.normal(0, 1), np.random.normal(0, 1), np.random.normal(0, 1)
```

Comment:
Could you please add a docstring about the purpose of this function? Is it to init the dataframe with values for `best_training_dice` etc?

Reply:
It was just to generate fake values to avoid running `ivadomed` when testing the script 😛